employee-management-client/src/styles.scss

```diff
@@ -1 +1,83 @@
 /* You can add global styles to this file, and also import other style files */
+
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
+
```

Comment:
Keep only those styling in style file which will remain same throughout the application. Component specific styling should in component itself

Reply:
This style will be used for all the components for now.